src/templates/app/user/settings/professional.jsx

```diff
@@ -19,7 +19,8 @@ const Professional = () => (
                 <p>{it.L('You are categorised as a professional client.')}</p>
             </div>
             <div className='invisible' id='rejected'>
-                <p>{it.L('Your request to be treated as a professional client has been rejected. Please check your inbox for more details. Your account remains categorised as a retail client account.')}</p>
+                <p>{it.L('Your request to be treated as a  professional is <strong>not approved</strong>.')}</p>
+                <p>{it.L('Please check your inbox for details. Your account remains under the retail category.')} <br /> {it.L(' You are welcome at any time if you\'d like to reapply as a professional client.')} </p>
```

Comment:
extra space before/after br tag and before closing p tag
please don't put a space in translation string. use `nbsp;` before it instead.